js/apps/admin-ui/src/realm-settings/security-defences/BruteForceDetection.tsx

```diff
@@ -155,6 +158,13 @@ export const BruteForceDetection = ({
               bruteForceMode ===
                 BruteForceMode.PermanentAfterTemporaryLockout) && (
               <>
+                <SelectControl
+                  name="bruteForceStrategy"
+                  label={t("bruteForceStrategy")}
+                  labelIcon={t("bruteForceStrategyHelp")}
+                  controller={{ defaultValue: bruteForceStrategyTypes[0] }}
+                  options={bruteForceStrategyTypes}
```

Comment:
Wouldn't it be nicer to have labels for these values?
```suggestion
                  options={bruteForceStrategyTypes.map((key) => ({ key, value: t(b) }))}
```
Or maybe even without this constant:
```javascript
options={[
            { key: "MULTIPLE", value: t("bruteForceStatedyMultiple") },
            { key: "LINEAR", value: t("bruteForceStatedyLinear") },

Reply:
@edewit done.